charts/dbildungs-iam-server/templates/telemetry-deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "common.names.name" . }}-telemetry
+  namespace: {{ template "common.names.namespace" . }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+    app.kubernetes.io/component: server-telemetry
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "common.names.name" . }}
+      app.kubernetes.io/component: server-telemetry
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        {{- include "common.labels" . | nindent 8 }}
+        app.kubernetes.io/component: server-telemetry
+    spec:
+      containers:
```

Comment:
I strongly doubt we need that. The collection should be done by SRE. Just providing the endpoints in the main app should be enough in the cluster. Just add a service that makes them publicly available